docs/configuration.md

```diff
@@ -209,10 +211,10 @@ Apart from these, the following properties are also available, and may be useful
   <td>(none)</td>
   <td>
     Set a special library path to use when launching the driver JVM.
-    
+
```

Comment:
Not a big deal but I see a fair bit of white-space-only change in the diff. It makes it a little harder to see what's changed. I don't know how much it's possible or worth fixing by hand but might check your IDE settings and not strip trailing spaces.